includes/class-sensei-course.php

```diff
@@ -4043,7 +4043,21 @@ public function get_average_days_to_completion() {
 		// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.PreparedSQL.NotPrepared, WordPress.DB.DirectDatabaseQuery.NoCaching -- Performance improvement.
 		return (float) $wpdb->get_var( $query );
 	}
-}
+
+	/**
+	 * Determines if course archive page has content.
+	 *
+	 * @since $$next-version$$
+	 * @return bool
+	 */
+	public function is_course_archive_page_has_content() {
```

Comment:
I find this function name to be a bit misleading, as it actually checks if the page has the query block, not if it has any 'ol content. What do you think about renaming it to something like `course_archive_page_has_query_block` and updating the description accordingly? I don't think we need the `is` because there's already a `has`. 🙂

Reply:
BTW, I think this is the last thing. 😉

Reply:
Hahah 😄 Thanks! Updated here https://github.com/Automattic/sensei/pull/6354/commits/37ff7776e3351a06f61f7cfbcf8c1aba5b804e0f